packages/flutter_tools/lib/src/compile.dart

```diff
@@ -282,10 +272,33 @@ class KernelCompiler {
         toMultiRootPath(dartPluginRegistrantFileUri, _fileSystemScheme, _fileSystemRoots, _fileSystem.path.separator == r'\');
     }
 
-    final List<String> command = <String>[
-      engineDartPath,
-      '--disable-dart-dev',
-      frontendServer,
+    late final List<String> commandToStartFrontendServer;
```

Comment:
nit, this can actually be final since the compiler can prove its always assigned before referenced:

```suggestion
    final List<String> commandToStartFrontendServer;
```